packages/runtime/src/widgets/Form/Checkbox.tsx

```diff
@@ -36,6 +46,7 @@ export const CheckboxWidget: React.FC<CheckBoxProps> = (props) => {
 
   return (
     <EnsembleFormItem valuePropName="checked" values={values}>
+      {leadingContent}
```

Comment:
Does this break the form submission?

Reply:
Yup, when we used the leadingContent, the checkbox checked stat is not updating.
so if we are not using the leadingContent, should I remove the leadingText prop from the definition ?